src/components/Global/RadioButton/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './RadioButton.module.scss';
+
+type DIFFICULTY = 'easy' | 'medium' | 'hard';
+
+type RadioButtonProps = {
+  id: string /*radioEasy*/;
+  name: string /*filter*/;
+  difficulty: DIFFICULTY;
+  checked: boolean;
+  setChecked: React.Dispatch<React.SetStateAction<boolean>>;
+};
+const getLabelColor = (difficulty: DIFFICULTY, isChecked: boolean) => {
+  return isChecked ? styles[`${difficulty}Checked`] : styles[`${difficulty}`];
```

Comment:
Even though this is a beautiful 1-line code, it creates a dependency between the name of the `DIFFICULTY` types and the names of the classes that I don't think we should have.

So I think the move is:
```ts
if(difficulty === 'easy') return isChecked ? styles.easyChecked : styles.easy;
// repeat for 'medium' and 'hard'
```

Reply:
Is dependency sth we want to reduce? If yes, why is that?
In this case, if we use if-statements, the value of label colour also depends on the value of difficulty, so what's the difference?